<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="USMUMDSINGHBHU5" ignored="0" name="Android One M8" tests="1" failures="1" timestamp="24 Feb 2019 16:11:23 GMT" time="3.346" errors="0">
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.demo.perfecto.Demo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.demo.perfecto.Demo.closeApp(Demo.java:124)
at com.demo.perfecto.Demo.afterClass(Demo.java:79)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="runTest" time="0.014" classname="com.demo.perfecto.Demo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.demo.perfecto.Demo.returnElement(Demo.java:130)
at com.demo.perfecto.Demo.performCalculation(Demo.java:107)
at com.demo.perfecto.Demo.runTest(Demo.java:73)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- runTest -->
</testsuite> <!-- Android One M8 -->
